NitroxServer/GameLogic/PlayerManager.cs

```diff
@@ -82,6 +82,11 @@ public List<Player> GetPlayers()
                 reservations.Add(reservationKey, playerContext);
                 assetPackage.ReservationKey = reservationKey;
 
+                if (ConnectedPlayers().Count > 0 && reservedPlayerNames.Count == 1)
```

Comment:
The additional `reservedPlayerNames.Count == 1` seems to be redundant.

Reply:
I did this because to prevent when having multiple players connect that it keeps starting the timer

Reply:
Can do ConnectedPlayers().Count == 1 then right?